ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -506,6 +506,20 @@ conformance_test(
     name = "conformance-test-in-memory-fan-out",
     ports = [6865],
     server = ":app",
+    server_args = [
+        "--contract-id-seeding=testing-weak",
+        "--participant participant-id=example,port=6865,ledger-api-transactions-buffer-max-size=100000",
+        "--buffered-ledger-api-streams",
+    ],
+    test_tool_args = [
+        "--verbose",
+    ],
+)
+
+conformance_test(
+    name = "conformance-test-in-memory-fan-out-tiny-buffer",
```

Comment:
duplicated conformance test suits for in memory fan out to have both scenarios:
* with huge buffer size, to test usage entirely from buffers `conformance-test-in-memory-fan-out`
* with tiny buffer size, to test usage with DB plus buffers usage `conformance-test-in-memory-fan-out-tiny-buffer`